pyqtorch/circuit.py

```diff
@@ -215,6 +215,114 @@ def sample(probs: Tensor) -> Counter:
             return list(map(lambda p: sample(p), probs))
 
 
+class DropoutQuantumCircuit(QuantumCircuit):
```

Comment:
Is there a need for another class or could this be integrated in `QuantumCircuit` ?

Reply:
This was a suggestion from @dominikandreasseitz, so that the QuantumCircuit class remained not overly complex.

Reply:
I agree with designing this on a separate class. All methods here are too specific to put in the QuantumCircuit.